distribution/docs/src/main/jdocs/content/_plugins/event-processor-contents.adoc

```diff
@@ -0,0 +1,75 @@
+
+When an event matches a subscription's criteria the Standard Event Processor:
+
+* invokes each pre-delivery plugin on the metacard in the event.
+* invokes the Delivery Method's operation corresponding to the type of event being processed, e.g., created operation for the creation of a metacard.
+
+===== Installing the Event Processor
+
+The Event Processor is installed by default with a standard installation in the ${ddf-catalog} application.
+
+===== Configuring the Event Processor
+
+The Event Processor has no configurable properties.
+
+===== Event Processing and Notification
+
+As metacards are created, updated, and deleted, the Catalog's Event Processor is invoked (as a post-ingest plugin) for each of these events.
+TheEvent Processor applies the filter criteria for each registered subscription to each of these ingest events to determine if they match the criteria.
+If an event matches a subscription's criteria, any pre-delivery plugins that are installed are invoked, the subscription's Delivery Method is retrieved, and its operation corresponding to the type of ingest event is invoked. 
+For example, the DeliveryMethod's `created()` function is called when a metacard is created.
+The Delivery Method's operations subsequently invoke the corresponding operation in the client's event consumer service, which is specified by the callback URL provided when the Delivery Method was created.
+
+.Available Event Processors
+* <<_event_processor,Standard Event Processor>>
+* <<_fanout_event_processor,Fanout Event Processor>>
+
+====== Using ${branding} Implementation
+
+If applicable, the implementation of `Subscription` that comes with ${branding} should be used.
+It is available at `ddf.catalog.event.impl.SubscriptionImpl` and offers a constructor that takes in all of the necessary objects.
+Specifically, all that is needed is a `Filter`, `DeliveryMethod`, `Set<String>` of source IDs, and a `boolean` for enterprise.
+
+The following is an example code stub showing how to create a new instance of Subscription using the ${branding} implementation. 
+
+.Creating a Subscription
+[source,java,linenums]
+----
+// Create a new filter using an imported FilterBuilder
+Filter filter = filterBuilder.attribute(Metacard.ANY_TEXT).like().text("*");
+ 
+// Create a implementation of Delivery Method
+DeliveryMethod deliveryMethod = new MyCustomDeliveryMethod();
+ 
+// Create a set of source ids
+// This set is empty as the subscription is not specific to any sources
+Set<String> sourceIds = new HashSet<String>();
+ 
+// Set the isEnterprise boolean value
+// This subscription example should notifications from all sources (not just local)
+boolean isEnterprise = true;
+
+Subscription subscription = new SubscriptionImpl(filter, deliveryMethod, sourceIds,isEnterprise);
+----
+
+===== Usage Limitations of the Event Processor
+
+The Standard Event processor currently broadcasts federated events and should not.
```

Comment:
❓ If this is a known issue, should it reference a DDF ticket for tracking purposes? Maybe also a TODO/FIXME with the ticket number in the documentation source?